cloudrun-malware-scanner/deploy.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Check if the environment value is provided as a command-line argument
+if [ $# -ne 3 ]; then
+    echo -e "Usage: $0 <env> <service_name> <service_account_name>"
+    echo -e "Possible values for <env> are: [pre-qa, qa, stg, prod]"
+    exit 1
+fi
+
+# Set the environment value from the command-line argument
+env_value="$1"
+service_name="$2"
+service_account_name="$3"
+unscanned_bucket_name="dapi-files-$env_value-unscanned"
+region="us-central1"
+
+# Input JSON file with "<env>"
+input_file="config.json.tmpl"
+
+# Output JSON file with replaced value
+output_file="config.json"
+
+# Replace "<env>" with the actual value and write to the output file
+sed "s/<env>/$env_value/g" "$input_file" > "$output_file"
+
+echo "Replacement complete. Output written to $output_file"
+
+gcloud beta run deploy "$service_name" \
```

Comment:
Cloud run is still in `beta`?

Reply:
Googles docs are out of date which still include `beta`. I'm testing without and seems to work ok so I'll remove beta here.